guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -49,7 +49,13 @@ endif::[]
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts* > *Register Host*.
 . Enter the details for how you want the registered host to be configured.
+* If you select a host group from the *Host Group* list, the *Operating system*
 ifdef::katello,satellite,orcharhino[]
+, *Activation Keys*
+endif::[]
+and *Lifecycle environment* fields inherit their values from the host group.
```

Comment:
@rh-max This is how the sentence is rendered:

![Screenshot From 2025-01-17 10-45-10](https://github.com/user-attachments/assets/a30b5e8f-f038-432c-a0ce-b055d667f81c)

Do you see it? How it might not be clear where the "If [...]" condition ends and the list of values start? How about something like this:

```suggestion
* If you select a host group from the *Host Group* list, the following fields inherit their values from the host group:
+
* *Operating system*
ifdef::katello,satellite,orcharhino[]
* *Activation Keys*
endif::[]
* *Lifecycle environment*
```